tests/unit/amo/components/TestApp.js

```diff
@@ -22,12 +22,18 @@ import {
   createContextWithFakeRouter,
   dispatchClientMetadata,
   dispatchSignInActions,
-  fakeCookie,
+  fakeCookies,
   fakeI18n,
   shallowUntilTarget,
   userAuthToken,
 } from 'tests/unit/helpers';
 
+// Skip `withCookies` HOC since Enzyme does not support the React Context API.
+// See: https://github.com/mozilla/addons-frontend/issues/6839
+jest.mock('react-cookie', () => ({
```

Comment:
This seems fine as a temporary hack but why not define it once in [`setup.js`](https://github.com/mozilla/addons-frontend/blob/master/tests/setup.js)? From a quick test on my end, it should work.

Reply:
Actually, that breaks lots of tests. It is better to mock when needed, not everywhere..